public/views/_test.html

```diff
@@ -353,7 +344,13 @@
                 }
             }
 
-            let results = await runWebTests(testFiles);
+            let results = {};
+
+            if (integrity) {
+                results = await codi.runWebTests(testFiles);
+            } else {
+                results = await codi.runWebTestFunction(_mappTest.coreTest);
```

Comment:
This webTestFunction might be better used on each test function in the test bundle to try aid parallel testing in the future.